src/L1ERC20Bridge.sol

```diff
@@ -76,17 +76,17 @@ contract L1ERC20Bridge is L1VotePool, WormholeSender {
   function receiveWormholeMessages(
     bytes memory payload,
     bytes[] memory additionalVaas,
-    bytes32 callerAddr,
+    bytes32 sourceAddress,
     uint16 sourceChain,
```

Comment:
Yes, the `receiveWormholeMessage` is the function the cross-chain relayer is calling. Other contracts on others chains could be sending messages causing the `sourceChain` to be different. And the `sourceChain` in the constructor is specifying the `sourceChain` for sending a message cross-chain and not receiving it cross chain.

Reply:
Got it, thanks. This can be tracked/done separately, but probably worth adding natspec and/or changing the variable names to disambiguate the two—IMO it's confusing that they have the same variable name but mean different things. Could imagine that `sourceChain` in the constructor means "receiveWormholeMessages can only receive messages from that source chain"

Reply:
Food call https://github.com/ScopeLift/l2-flexible-voting/issues/47#issue-1893127860